collect_app/src/main/java/org/odk/collect/android/events/MediaPlayerRxEvent.java

```diff
@@ -19,10 +19,14 @@
 public class MediaPlayerRxEvent extends RxEvent {
 
     private final EventType eventType;
-    private final int playerTag;
+    private int playerTag;
 
     public enum EventType {
-        PLAYING_STARTED, PLAYING_STOPPED, PLAYING_COMPLETED
+        PLAYING_STARTED, PLAYING_STOPPED, PLAYING_COMPLETED, STOP_PLAYING
```

Comment:
The first three events are events broadcast by a player to announce its state. The last one is an event broadcast to a player to change its state. Is it typical to mix events from different sources?